.gitignore

```diff
@@ -182,3 +182,9 @@ $RECYCLE.BIN/
 
 # build folders under Linux
 Microsoft.WindowsAzure.Storage/build.*/
+
+# ==========================
+# OSX detritus
+# ==========================
+
+*.DS_Store
```

Comment:
In githubs template for OSX they have it as *.DS_Store. I'm pretty sure .DS_Store without the star didn't work.